packages/react/src/components/DotcomShell/README.md

```diff
@@ -0,0 +1,82 @@
+# DotcomShell
+
+> The DotcomShell component includes the `Masthead`, `Footer`, and `Content`
+> components, all wrapped in a UI shell using Carbon's grid.
+
+## Getting started
+
+The `DotcomShell` uses Carbon's `Content` component. All user-provided content
+should be placed here. This can be anything from HTML to a React component.
+Here's a quick example to get you started.
+
+### Content
+
+```javascript
+import React from 'react';
+import { Button, StructuredListBody } from 'carbon-components-react';
+
+const content = (
+  <>
+    <p>Your content here (probably include something nicer than this! 😄)</p>
```

Comment:
Hm, not sure what you mean. The content is already broken out for Storybook. Not as a component, but clear enough (I think) as an example of how to include content. It can be a component, plain HTML, etc. I think it's pretty clear here? What do you suggest we use as an example?

Reply:
Maybe I'm misunderstanding something, but the way I'm looking at this documentation is what's the bare minimum to get the adopter started. A content component isn't exactly necessary, and according to the instructions already included within the `DotcomShell`. I'm afraid it might be adding noise to the instructions. The important part here is the `DotcomShell`. If we remove the content component we move it up higher up more prioritizing the meat in the content while also shortening the amount of information to parse. We could simplify it to something like this...

> ## Getting started
> 
> The `DotcomShell` uses Carbon's `Content` component. All user-provided content should be placed here. This can be anything from HTML to a React component. Here's a quick example to get you started.
> 
> ```javascript
> import React from 'react';
> import { DotcomShell } from '@carbon/ibmdotcom-react';
> 
> import '@carbon/ibmdotcom-styles/scss/components/dotcom-shell/_dotcom-shell.scss';
> 
> function App() {
>   return (
>     <DotcomShell>
>       <p>Add your content, or other components here!</p>
>     </DotcomShell>
>   );
> }
> ```

Reply:
Ok sure

Reply:
ok sure